src/www/status_dhcpv6_leases.php

```diff
@@ -307,9 +307,19 @@ function parse_duid($duid_string)
         }
     }
 
-    if ($_GET['order']) {
+    $order = ( $_GET['order'] ) ? $_GET['order'] : 'ip';
```

Comment:
this looks fishy -- some pages have weak handling of input arguments and their defaults

Reply:
hmm, ok it emulates the dhcp v4 leases page. now we have two problems :D

Reply:
Yes, I copied it from the DHCPv4 leases page. Suggested a more secure way for both pages in commit 43427bb.